src/storage/mvcc/reader/reader.rs

```diff
@@ -140,7 +140,7 @@ impl<S: Snapshot> MvccReader<S> {
         }
 
         let cursor = self.write_cursor.as_mut().unwrap();
-        let ok = cursor.near_seek(&key.clone().append_ts(ts), &mut self.statistics.write)?;
+        let ok = cursor.seek(&key.clone().append_ts(ts), &mut self.statistics.write)?;
```

Comment:
Why not  near_seek?

Reply:
Because the keys which we are ready to prewrite are not adjacent, we may call too many times `next` in `near_seek`.
I will add a new interface to use `seek` instead of `near_seek`.